### PR TITLE
Add couch views for active reports

### PR DESCRIPTION
**Pull request by mkangia**
follow up from https://github.com/dimagi/commcare-hq/pull/21977#issuecomment-425923445

total 1576 inactive configs and 1734 active as of now on prod.

table_id in the view can help filter [here](https://github.com/dimagi/commcare-hq/blob/03885be50bf6a251116b4a150aee32e02723df5c/corehq/apps/userreports/pillow.py#L107)

### CODE REVIEW

**Comment by mkangia** [comment]
What should be the process of deploying this?
setup_release and run some preindex command?

File: corehq/couchapps/phone_numbers/views/verified_number_by_domain/map.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function(doc) {
     if(doc.doc_type == "VerifiedNumber") {
-        emit([doc.domain, doc.owner_doc_type, doc.owner_id], null);
+        emit([doc.domain, doc.owner_doc_type, doc.table_id, doc.engine_id], null);
```
Comment (snopoke) [inline corehq/couchapps/phone_numbers/views/verified_number_by_domain/map.js]: did you mean to change this view?

**Review by snopoke — COMMENTED** [review]
how will 'engine_id` help?

File: corehq/couchapps/phone_numbers/views/verified_number_by_domain/map.js (hunk 1 above)
Comment (mkangia) [reply corehq/couchapps/phone_numbers/views/verified_number_by_domain/map.js]: Ya

Comment (mkangia) [reply corehq/couchapps/phone_numbers/views/verified_number_by_domain/map.js]: sorry no. Thanks for catching this.

**Comment by mkangia** [comment]
@snopoke i sawa that the only thing we finally need is the engine_id
https://github.com/dimagi/commcare-hq/blob/mk-add-active-reports-view/corehq/apps/userreports/sql/connection.py#L15
So was just trying to add things we could use now/later

Nvm,
Saw that again and looks like there is an __init__ in super class that needs more.

removed engine_id. retained table_id

**Comment by mkangia** [comment]
Thanks @emord .
You mean after the deploy is done
merge this branch
and then setup_release and `preindex_everything` (is the command to run)?

**Comment by emord** [comment]
> You mean after the deploy is done
> merge this branch
> and then setup_release and preindex_everything (is the command to run)?

Correct. Preindexes on couch generally block future deploys while couch is building the index. I think that this one should be quick, but don't actually know how large that db is

**Comment by mkangia** [comment]
I ran this on production by setting up a release with this branch and then running `preindex_everything`. It ran in like 15 mins.
Checked with Simon that other environments should be able to get this within deploy itself.

Build failed due to an unrelated reason so would merge post a successful run.

@emord @snopoke 
also, assumed this was a safe change to reindex to not miss the opportunity of the weekend, though still needs a review/approval.

**Review by snopoke — APPROVED** [review]

**Pull request merged**